@hubblecommerce/hubble/core/components/checkout/PaymentMethods.vue

```diff
@@ -102,6 +102,11 @@ export default {
                 context.emit('processing', false);
                 context.emit('payment-error', false);
                 context.emit('payment-changed', currentMethodObj.value);
+                $nuxt.$emit('checkout-payment-method-changed', {
+                    checkout: {
+                        actionField: { step: 2, option: currentMethodObj.value.name }
```

Comment:
Remove checkout step to keep data apstract: https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce#checkoutstep